docs/extensions/mysql_scanner.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: docu
+title: MySQL Scanner Extension
+---
+
+The `mysql` extension allows DuckDB to directly read data from a running MySQL instance. The data can be queried directly from the underlying MySQL tables, or read into DuckDB tables.
+
+## Usage
+
+To make a MySQL database accessible to DuckDB, use the `mysql_attach` command:
+
+TODO: edit the rest
+
+```sql
```

Comment:
Perhaps just copy the [`README`](https://github.com/Mytherin/mysql_scanner) from here?

Reply:
sure, thx